### DISCUSSION TITLE
[Translate] Is ` letter in docs means <code> ?

### DISCUSSION THREAD

**Discussion by tamagoez** [saadeghi/daisyui#598] [Q&A] [answered]
I found ` letter in https://daisyui.com/docs/use/?lang=en#:~:text=component%20classes%20like-,%60btn%60%2C%20%60card%60,-%2C%20etc. .
Should I translate it to \<code> ?

**Comment by saadeghi** [comment, marked as answer]
Yes, Markdown was converting \`string\` to `<code>string</code>` but now that strings are coming from the `Translate` function, that doesn't work anymore.

**Reply by tamagoez** [reply]
Thanks
